tests/classes/notification/PKPNotificationManagerTest.php

```diff
@@ -354,13 +354,15 @@ private function getFixtureCreateNotificationSendEmail($expectedNotification)
             ->will($this->returnValue($mailTemplateMock));
 
         // Register a UserDao stub to return the test user.
-        $userDaoStub = $this->getMockBuilder(UserDAO::class)
-            ->setMethods(['getById'])
-            ->getMock();
-        $userDaoStub->expects($this->any())
-            ->method('getById')
-            ->will($this->returnValue($testUser));
-        DAORegistry::registerDAO('UserDAO', $userDaoStub);
+        /*        $userDaoStub = $this->getMockBuilder(\PKP\user\DAO::class)
+                    ->setMethods(['getById'])
+                    ->getMock();
+                $userDaoStub->expects($this->any())
+                    ->method('getById')
+                    ->will($this->returnValue($testUser));*/
```

Comment:
Did you intend to leave this commented block in?

Reply:
Fixed, thanks!